example/package.json

```diff
@@ -2,41 +2,38 @@
   "private": true,
   "scripts": {
     "start": "babel-node ./server.js",
-    "build": "relay-compiler --src ./ts/ --schema ./data/schema.graphql --language typescript --artifactDirectory ./ts/__relay_artifacts__",
+    "build":
+      "relay-compiler --src ./ts/ --schema ./data/schema.graphql --language typescript --artifactDirectory ./ts/__relay_artifacts__",
     "update-schema": "babel-node ./scripts/updateSchema.js",
-    "lint": "eslint ./js",
+    "lint": "tslint --project tsconfig.json",
     "type-check": "tsc --noEmit --pretty"
   },
   "resolutions": {
-    "graphql": "^0.13.1",
-    "relay-compiler-language-typescript": "file:../relay-compiler-language-typescript-1.1.0-alpha.1.tgz",
-    "relay-compiler": "https://github.com/alloy/relay/releases/download/v1.6.1-plugin.1/relay-compiler-1.6.1-plugin.1.tgz",
-    "graphql-compiler": "https://github.com/alloy/relay/releases/download/v1.6.1-plugin.1/graphql-compiler-1.6.1-plugin.1.tgz",
-    "relay-runtime": "https://github.com/alloy/relay/releases/download/v1.6.1-plugin.1/relay-runtime-1.6.1-plugin.1.tgz",
-    "babel-plugin-relay": "https://github.com/alloy/relay/releases/download/v1.6.1-plugin.1/babel-plugin-relay-1.6.1-plugin.1.tgz",
-    "react-relay": "https://github.com/alloy/relay/releases/download/v1.6.1-plugin.1/react-relay-1.6.1-plugin.1.tgz"
+    "relay-compiler-language-typescript":
+      "file:../relay-compiler-language-typescript-v1.1.1-alpha.2.tgz"
   },
   "dependencies": {
+    "@babel/core": "^7.0.0-0",
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
-    "babel-plugin-relay": "https://github.com/alloy/relay/releases/download/v1.6.1-plugin.1/babel-plugin-relay-1.6.1-plugin.1.tgz",
+    "babel-plugin-relay": "^2.0.0-rc.1",
     "babel-plugin-transform-runtime": "^6.12.0",
     "babel-preset-es2015": "^6.13.2",
     "babel-preset-react": "^6.11.1",
     "babel-preset-stage-0": "^6.5.0",
     "babel-runtime": "^6.26.0",
     "classnames": "2.2.5",
     "express": "^4.15.2",
-    "express-graphql": "^0.6.4",
-    "graphql": "^0.13.1",
-    "graphql-compiler": "https://github.com/alloy/relay/releases/download/v1.6.1-plugin.1/graphql-compiler-1.6.1-plugin.1.tgz",
-    "graphql-relay": "^0.5.1",
+    "express-graphql": "^0.7.1",
+    "graphql": "^14.0.2",
+    "graphql-compiler": "^2.0.0-rc.1",
+    "graphql-relay": "^0.5.5",
     "prop-types": "^15.6.2",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "react-relay": "https://github.com/alloy/relay/releases/download/v1.6.1-plugin.1/react-relay-1.6.1-plugin.1.tgz",
+    "react-relay": "^2.0.0-rc.1",
```

Comment:
does we need to upgrade react and react-dom as well?

related to this https://github.com/facebook/relay/issues/2565#issuecomment-447805467

see https://github.com/facebook/relay/releases/tag/v2.0.0-rc.1